error.go

```diff
@@ -92,3 +92,24 @@ func (e *RequestError) Error() string {
 func (e *RequestError) Unwrap() error {
 	return e.Err
 }
+
+func (e *ErrorResponse) UnmarshalJSON(data []byte) (err error) {
+	var rawMap map[string]json.RawMessage
+	err = json.Unmarshal(data, &rawMap)
+	if err != nil {
+		return
+	}
+	if _, ok := rawMap["error"]; !ok {
+		e.Error = &APIError{
+			Code: "unknown",
+		}
+		if _, okMessage := rawMap["message"]; okMessage {
+			if json.Unmarshal(rawMap["message"], &e.Error.Message) == nil {
```

Comment:
@zjy282 The activityId gets lost here.

Reply:
@vvatanabe Thanks for pointing out the error, I'll fix it next week

Reply:
> @zjy282 The activityId gets lost here.

@vvatanabe done!